src/main/java/org/fundacionjala/coding/jorge/SortTheInnerContent.java

```diff
@@ -0,0 +1,44 @@
+package org.fundacionjala.coding.jorge;
+
+import java.util.Arrays;
+import java.util.StringJoiner;
+
+/**
+ *
+ */
+public final class SortTheInnerContent {
+    public static final int LIMIT = 3;
+    /**
+     *
+     */
+    private SortTheInnerContent() {
+    }
+
+    /**
+     * @implSpec tamMin values mini
+     * @param words given type String
+     * @return sb given type StringBuilder
+     */
+    public static String sortTheContent(final String words) {
+        StringJoiner sj = new StringJoiner(" ");
+        String[] wordsArr = words.split(" ");
+        for (String word : wordsArr) {
+            sj.add(word.length() > LIMIT ? word.substring(0, 1)
+                    .concat(reverse(word.substring(1, word.length() - 1).toCharArray()))
+                    .concat(word.substring(word.length() - 1)) : word);
+        }
+        return sj.toString();
+    }
+
+    /**
+     * @param chars given type String
+     * @return sb given type String
+     */
+    private static String reverse(final char[] chars) {
+        char[] c = chars;
+        StringBuilder sb = new StringBuilder();
+        Arrays.sort(c);
+        sb.append(String.valueOf(c));
+        return String.valueOf(sb.reverse());
```

Comment:
this may be refactored like this.
```java
    private static String reverse(final char[] chars) {
        Arrays.sort(chars);
        return new StringBuilder(String.valueOf(chars)).reverse().toString();
    }
```